x/concentrated-liquidity/incentives_test.go

```diff
@@ -2205,6 +2211,7 @@ func (s *KeeperTestSuite) TestQueryAndCollectIncentives() {
 				sdk.NewDec(0),
 				sdk.NewDec(9981),
 				sdk.NewDec(1),
+				sdk.NewDec(778212931834),
```

Comment:
Can we add in line comments on why || how we are getting these numbers? Would be helpful in coming back to these test cases

Reply:
This specific one is just a random number to test non-default liquidity amounts - will add comments to clarify